src/views/sign-up-view/index.tsx

```diff
@@ -0,0 +1,120 @@
+// Third-party imports
+import Link from 'next/link'
+
+// HashiCorp imports
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import { IconCheckCircle16 } from '@hashicorp/flight-icons/svg-react/check-circle-16'
+import { IconUserPlus16 } from '@hashicorp/flight-icons/svg-react/user-plus-16'
+import InlineSvg from '@hashicorp/react-inline-svg'
+
+// Global imports
+import useAuthentication from 'hooks/use-authentication'
+import BaseNewLayout from 'layouts/base-new'
+import ErrorView from 'views/error-view-switcher'
+import Button from 'components/button'
+import Heading from 'components/heading'
+import Text from 'components/text'
+
+// Local imports
+import s from './sign-up.module.css'
+
+// Text content
+const TITLE = 'Create Your Account'
+const DETAILS = [
+	'Use your account across HashiCorp products',
+	'Track your progress in longer tutorials',
+	'Bookmark tutorials you need to reference regularly',
+]
+const SIGN_UP_BUTTON_TEXT = 'Sign Up'
```

Comment:
howdy @ashleemboyer ! great work on this sign up page. there was a big push a while back to standardize text casing to sentence case in HCP. just wanted to mention it here. "Sign Up" => "Sign up".

do y'all have a Figma board where i can see more of the designs for the sign up pages?

Reply:
Hi @cbfx! Thanks for calling this out. Here's a link to the Figma for this page: https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=11365%3A114999.

cc-ing @braydenlove and @heitzke for visibility

Reply:
Thanks, will update!

Reply:
@ashleemboyer no problem! nice to meet y'all, btw!

Reply:
Nice to meet you @cbfx!! 👋🏻